tests/framework/installer/ceph_manifests.go

```diff
@@ -816,20 +816,6 @@ metadata:
   labels:
     operator: rook
     storage-backend: ceph
-aggregationRule:
```

Comment:
It seems like the `ceph_manifests.go` are changing less than the manifests themselves. Should there be these changes applied for the upgrade test also?

Reply:
Yes, I do need to update the upgrade test as well

Reply:
@BlaineEXE The upgrade test is updated now as well